examples/with-externals/next.config.js

```diff
@@ -0,0 +1,26 @@
+const webpack = require("webpack")
+
+// Update these to match your package scope name.
+const internalNodeModulesRegExp = /@zeit(?!.*node_modules)/
+const externalNodeModulesRegExp = /node_modules(?!\/@zeit(?!.*node_modules))/
+
+module.exports = {
+  webpack: (config, { dev, isServer, defaultLoaders }) => {
+    config.resolve.symlinks = false
+    config.externals = config.externals.map(external => {
+      if (typeof external !== "function") return external
+      return (ctx, req, cb) => (internalNodeModulesRegExp.test(req) ? cb() : external(ctx, req, cb))
+    })
+    config.module.rules.push({
+      test: /\.+(js|jsx)$/,
+      loader: defaultLoaders.babel,
```

Comment:
Does this respect my local babel config? Trying this out but wouldn't process stage-2.

Reply:
Never mind, it does work!

Reply:
Yep `defaultLoaders.babel` is provided by next to be integrated into plugins / next.config.js. It's the same babel config used to configure `.js` transpilation 🙏

Reply:
I can say that this `next.config.js` does indeed work.  I took what's here and tested in my own project (Lerna based).